.travis.yml

```diff
@@ -6,7 +6,7 @@ before_install:
 - pip install -U pip wheel setuptools
 
 install:
-- pip install cookiecutter
+- pip install --no-use-pep517 cookiecutter
```

Comment:
is this needed?

Reply:
I guess not - just so used to it for aiida, hehe - will remove it, thanks.